src/services/deleteItemFromFirebase.tsx

```diff
@@ -1,18 +1,28 @@
+export enum Categories {
+  SPELLS = 'spells',
+  WEAPONS = 'weapons',
+  ITEMS = 'items',
+}
+
+export type CategoryTypes =
+  | Categories.SPELLS
+  | Categories.WEAPONS
+  | Categories.ITEMS;
```

Comment:
Bardzo ladnie ;d ale jeszcze jest sporo miejsc gdzie mozna to dodać (i ogolnie zapisałbym to gdzies w globalnym folderze types/CategoryTypes.ts